android_jni/avifandroidjni/src/main/jni/libavif_jni.cc

```diff
@@ -0,0 +1,150 @@
+#include <android/bitmap.h>
+#include <android/log.h>
+#include <jni.h>
+#include <string.h>
+
+#include "avif/avif.h"
+
+#define LOG_TAG "avif_jni"
```

Comment:
Nit/optional: Maybe "avif_android_jni"? Would that be too long?

Reply:
"android" is implied since it's part of android jni :)